test/EndToEnd.Tests/GivenDotNetUsesMSBuild.cs

```diff
@@ -3,20 +3,16 @@
 
 [assembly: CollectionBehavior(DisableTestParallelization = true)]
 
-namespace Microsoft.DotNet.Tests.EndToEnd
+namespace EndToEnd.Tests
 {
-    public class GivenDotNetUsesMSBuild : SdkTest
+    public class GivenDotNetUsesMSBuild(ITestOutputHelper log) : SdkTest(log)
     {
-        public GivenDotNetUsesMSBuild(ITestOutputHelper log) : base(log)
-        {
```

Comment:
Thanks for removing this unnecessary constructors!

Reply:
So, this didn't remove it. It is just using primary constructor notation instead.